test/integration/job/job_test.go

```diff
@@ -1026,6 +1026,334 @@ func TestIndexedJob(t *testing.T) {
 	validateTerminatedPodsTrackingFinalizerMetric(t, 5)
 }
 
+// This test has 4 parts which execute in the following order:
+// 1) Create Indexed Job then succeed/fail some indexes
+// 2) Optional: scale up then succeed/fail some indexes
+// 3) Optional: scale down then succeed/fail some indexes
+// 4) Optional: scale back up again
+//
+// Parts 1-3 are only executed if a flag in the test case
+// indicates that they should do so. Therefore, each test
+// case can toggle them to test various scenarios involving
+// scaling up/down/scaling up again, etc.
+func TestElasticIndexedJob(t *testing.T) {
+	cases := map[string]struct {
+		// Initial job
+		initialParallelism                int
+		initialCompletions                int
```

Comment:
we can just assume the same value for all tests and remove those two parameters.